src/app/modules/did-book/components/did-book-form/did-book-form.component.ts

```diff
@@ -19,6 +19,7 @@ import { DidBookRecord } from '../models/did-book-record';
 export class DidBookFormComponent implements OnInit {
   @Input() did: string;
   @Input() label: string;
+  @Input() isDidBook: boolean;
```

Comment:
Maybe more about the context inside this component, not outside? isDidBook tells nothing about this component. 
I prefer to add here 2 inputs like:
```suggestion
  @Input() showClear: boolean;
  @Input() showCancel: boolean;
```
This tells more about behaviour in this component. What do you think?

Reply:
Thanks @dawidgil, made these changes